tests/functional/pod_and_daemons/test_mgr_enable_rook_backend_module.py

```diff
@@ -21,6 +21,7 @@
 @brown_squad
 @skipif_external_mode
 @skipif_ocs_version("<4.15")
+@skipif_ocs_version(">=4.18")
```

Comment:
can you check if it can be combined below way?
@skipif_ocs_version(["<4.15", ">4.17"])